src/main/java/spiffe/api/svid/SpiffeWorkloadStub.java

```diff
@@ -0,0 +1,71 @@
+package spiffe.api.svid;
+
+import io.grpc.*;
+import io.grpc.stub.StreamObserver;
+import spiffe.api.svid.SpiffeWorkloadAPIGrpc.SpiffeWorkloadAPIStub;
+
+import static spiffe.api.svid.Workload.*;
+
+/**
+ * Wraps and configures a SpiffeWorkloadAPIBlockingStub with a ManagedChannel
+ * and an ClientInterceptor to add the SECURITY HEADER
+ *
+ */
+class SpiffeWorkloadStub {
+
+    private static final String SECURITY_HEADER = "workload.spiffe.io";
+
+    private SpiffeWorkloadAPIStub workloadAPIAsyncStub;
+
+    /**
+     * Constructor
+     * @param spiffeEndpointAddress where the WorkloadAPI is listening
+     */
+    SpiffeWorkloadStub(String spiffeEndpointAddress) {
+        ManagedChannel managedChannel = SpiffeEndpointChannelBuilder.newChannel(spiffeEndpointAddress);
```

Comment:
can you add validations to spiffeEndpointAddress?

Reply:
i read below that we delegate validation to builder, can you add a comment here too?

Reply:
it's done